modules/storages/spec/features/create_and_delete_project_storage_spec.rb

```diff
@@ -65,6 +72,12 @@
     expect(page).to have_text('Enable a file storage')
     page.find('button[type=submit]').click
 
+    # ToDo: Error message here (in the browser):
+    # Project was attempted to be written but is not writable.
+    # Storage was attempted to be written but is not writable.
+    # Creator was attempted to be written but is not writable.
+    # Also appears when executing as :admin
+
```

Comment:
@fraber Here, we should cover the edge cases of errors. We want to cover that validation errors etc. trigger a error message. We do not need to cover all possible validation errors as that is already done in the contract spec.